src/vos/ilog.h

```diff
@@ -237,21 +240,39 @@ ilog_fetch(struct umem_instance *umm, struct ilog_df *root,
 void
 ilog_fetch_finish(struct ilog_entries *entries);
 
+/** Convenience structure for iterators */
+struct ilog_iter {
```

Comment:
This is unused.

Reply:
Thanks, I tried to remove the dead code.  I'll remove this in subsequent patch if I need a refresh